### ISSUE TITLE
Make toggle NMD button display as a switch you can turn off or on 

### ISSUE THREAD

**Issue by immattdavison** [immattdavison/NoMoreDomains#44]
Show on the button whether it is powered on or off and allow for it to be toggled accordingly

**Comment by Rutuj-Runwal** [comment]
The status currently is displayed as a separate text above the button?

Should it also be kept?

**Comment by immattdavison** [comment]
> The status currently is displayed as a separate text above the button?
> 
> Should it also be kept?

The issue there is the colour contrast and that it doesn't stay there after the popup is closed, I would like to have a proper toggle switch like the one shown in darkreader 

![image](https://user-images.githubusercontent.com/80921257/193472709-c0c7ca91-d918-4c20-b3da-6e2bdf3529b5.png)

**Comment by Rutuj-Runwal** [comment]
This is another extension that I have built if you ca, do scroll through the readme there you will see the toggle that I have implemented in the readme
Can we do something similar?
Link: https://github.com/Rutuj-Runwal/RR-Adblocker/blob/main/README.md

Or do you prefer the dark reader style.

**Comment by immattdavison** [comment]
Yes I like that! Maybe have the toggle turn red if it’s off and green if it’s on? On 2 Oct 2022, at 20:35, Rutuj Runwal ***@***.***> wrote:﻿
This is another extension that I have built if you ca, do scroll through the readme there you will see the toggle that I have implemented in the readme
Can we do something similar?
Link: https://github.com/Rutuj-Runwal/RR-Adblocker/blob/main/README.md
Or do you prefer the dark reader style.

**Comment by Rutuj-Runwal** [comment]
> Yes I like that! Maybe have the toggle turn red if it’s off and green if it’s on? On 2 Oct 2022, at 20:35, Rutuj Runwal ***@***.***> wrote:﻿
> This is another extension that I have built if you ca, do scroll through the readme there you will see the toggle that I have implemented in the readme
> Can we do something similar?
> Link: https://github.com/Rutuj-Runwal/RR-Adblocker/blob/main/README.md
> Or do you prefer the dark reader style.
> 
> —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

Okay sure if you want you can experiment with the toggle colors and design here

https://www.w3schools.com/howto/tryit.asp?filename=tryhow_css_switch

I will make the implementation logic till tomorrow till then you can fix the designs  and colors etc precisely🙂

**Comment by immattdavison** [comment]
> > Yes I like that! Maybe have the toggle turn red if it’s off and green if it’s on? On 2 Oct 2022, at 20:35, Rutuj Runwal _**@**_.***> wrote:﻿
> > This is another extension that I have built if you ca, do scroll through the readme there you will see the toggle that I have implemented in the readme
> > Can we do something similar?
> > Link: https://github.com/Rutuj-Runwal/RR-Adblocker/blob/main/README.md
> > Or do you prefer the dark reader style.
> > —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: _**@**_.***>
> 
> Okay sure if you want you can experiment with the toggle colors and design here
> 
> https://www.w3schools.com/howto/tryit.asp?filename=tryhow_css_switch
> 
> I will make the implementation logic till tomorrow till then you can fix the designs and colors etc precisely🙂

Ok so you can work on this tomorrow?

**Comment by Rutuj-Runwal** [comment]
Yes surely
Actually it's 1AM here so will do it in the morning It won't be possible now

**Comment by immattdavison** [comment]
> Yes surely Actually it's 1AM here so will do it in the morning It won't be possible now

That's all good with me! Thank you!
